constructor/main.py

```diff
@@ -89,6 +89,80 @@ def _win_install_needs_python_exe(conda_exe: str) -> bool:
     return results.returncode == 2
 
 
+# Validate frozen environments
+def validate_frozen_envs(info, exe_type, exe_version):
+    """Validate frozen environments.
+
+        Checks:
+        - No conflicts between freeze_base/freeze_env and extra_files for same environment
+        - conda-standalone 25.5.x is not used (has known issues)
+        - Warns if conda-standalone < 25.5.0 (frozen files will be ignored)
+    """
+    def get_frozen_env_from_path(dest: str | dict) ->set[str] | None:
+        """Extract environment name from frozen marker destination path.
+
+        Returns:
+            Set of environment names found in the path, or None if no frozen marker found.
+        """
+
+        def get_env(path: str) -> str | None:
+            parts = Path(path).parts
+            if parts == ("conda-meta", "frozen"):
+                return "base"
+            elif len(parts) == 4 and parts[0] == "envs" and parts[-2:] == ("conda-meta", "frozen"):
+                return parts[1]
+            return None
+
+        envs = set()
+
+        if isinstance(dest, str):
+            if env := get_env(dest):
+                envs.add(env)
+
+        if isinstance(dest, dict):
+            for path in dest.values():
+                if env := get_env(path):
+                    envs.add(env)
+
+        return envs if envs else None
```

Comment:
```suggestion
        return envs
```

You could just return an empty `set` if no frozen environments are found.